app/src/main/java/org/mozilla/fenix/compose/ListItemTabLarge.kt

```diff
@@ -104,7 +105,9 @@ private fun ListItemTabSurface(
     onClick: (() -> Unit)? = null,
     tabDetails: @Composable () -> Unit
 ) {
-    var modifier = Modifier.size(328.dp, 116.dp)
+    var modifier = Modifier
+        .size(328.dp, 116.dp)
+        .testTag("ListItemTabLarge")
```

Comment:
Looks like we'll have to add `testTag`s to all custom composables which are more complex than just a text / contentDescription.